app/src/main/java/com/sdp/swiftwallet/di/AuthenticatorModule.java

```diff
@@ -0,0 +1,23 @@
+package com.sdp.swiftwallet.di;
+
+import com.sdp.swiftwallet.domain.repository.AuthenticatorFirebaseImpl;
+import com.sdp.swiftwallet.domain.repository.SwiftAuthenticator;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.hilt.InstallIn;
+import dagger.hilt.components.SingletonComponent;
+
+/**
+ * Hilt Module to inject SwiftAuthenticator
+ */
+@Module
+@InstallIn(SingletonComponent.class)
+public class AuthenticatorModule {
+
+    @Provides
```

Comment:
is it normal to not have @Singleton tag here ?

Reply:
Yes, for modules you use @InstallIn(SingletonComponent.class)